docs/source/tutorial.rst

```diff
@@ -45,6 +45,7 @@ body of response reveals the information about the created tender: its internal
 `dateModified` datestamp stating the moment in time when tender was last
 modified. Pay attention to the `procurementMethodType`. Note that tender is
 created with `active.enquiries` status.
+Also, you can create test tender, just adding `"mode": "test"` field to the body of your request. This tender can be used for experimental purposes. Test mode can be set only when creating a new tender and can not be changed anymore.
```

Comment:
Also, you can create a test tender by simply adding `"mode": "test"` field to the body of your request. This tender can be used for experimental purposes. Test mode can be set only during creation of a new tender and can not be changed later.